controllers/dagora.controllers.js

```diff
@@ -1,7 +1,7 @@
-const { redisClient } = require('../lib/redisClient');
-const { generateAgoraThreadsRedisKey } = require('../util/generateRedisKeys');
+// const { redisClient } = require('../lib/redisClient');
+// const { generateAgoraThreadsRedisKey } = require('../util/generateRedisKeys');
 const { queryClientForThreadsByCategory, queryClientForRepliesByThreadId, queryClientForThreadById} = require('../helpers/queries');
-const { getCacheItemIndexFromList } = require('../helpers/cache');
+// const { getCacheItemIndexFromList } = require('../helpers/cache');
 const { MAX_CACHED_THREADS } = require('../constants');
```

Comment:
we are using constant ```MAX_CACHED_THREADS``` with redis only, this can be removed as well.

Reply:
Okay

Reply:
Removed